stdeb.cfg

```diff
@@ -1,4 +1,5 @@
 [colcon-powershell]
 Depends3: python3-colcon-core (>= 0.3.0)
+Recommends3: powershell
```

Comment:
I don't think `powershell` is a valid Debian package in any of the targeted Ubuntu distros. Can you provide a link to the Ubuntu package if there is one?

Reply:
`Powershell` require extra repository
https://docs.microsoft.com/fr-fr/powershell/scripting/setup/installing-powershell-core-on-linux?view=powershell-6#ubuntu-1604

Reply:
Then this dependency can't be specified here since it wouldn't be satisfiable if the user hasn't added this repo.